scripts/build.ts

```diff
@@ -411,8 +411,8 @@ export default async function (
             } else {
               obj[depName] = `github:${pkg.snapshotRepo}#${pkg.snapshotHash}`;
             }
-          } else if ((obj[depName] as string).match(/\b0\.0\.0\b/)) {
-            obj[depName] = (obj[depName] as string).replace(/\b0\.0\.0\b/, v);
+          } else if ((obj[depName] as string).match(/\b0\.0\.0-PLACEHOLDER\b/)) {
```

Comment:
Do we need to escape the `-` in this case?

```
\b0\.0\.0\-PLACEHOLDER\b
```

Reply:
Nope, the `-` is treated as a literal outside of a character class.